source/index.html.md

```diff
@@ -146,52 +142,98 @@ certain triggers may pass in.
   You can register as many triggers as you want, and you can even use the same function in multiple different triggers.
 </aside>
 
-## Trigger arguments
+## Chat Trigger
```

Comment:
Let's make this something like `The "chat" Trigger` so that it doesn't get confused with the name of the mod.